### PR TITLE
Add error docs to post interactions and undo adding a post body to mute

### LINKED ISSUE #33: Interactions API Endpoint Updates
- [x] "Hide" is really toggling hide (just as the UI does) so should have documentation changed to reflect that
- [x] Add Error codes for the endpoints
- [x] Add the payload data for the Report method

### PR DESCRIPTION
**Pull request by jhass**
Remove the mistakenly in ed4de6a702f19291538e4cc760ddd767d42ffd34 added request body, it was never implemented like this.

I think adding the error codes should also close #33 if I understood the issue and its state right.

### CODE REVIEW

File: _routes/post_interactions.md [hunk 1]
```diff
@@ -39,8 +39,11 @@ Status: 204 No Content
 
 | Status code | Error reason                               |
 | ----------- | ------------------------------------------ |
+| 404         | Post with provided guid could not be found |
 | 409         | Post has already been reported             |
+| 422         | Request invalid, such as a missing reason  |
 
+>>>>>>> Add error docs to post interactions and undo adding a post body to mute
```
Comment (skyschub) [inline _routes/post_interactions.md]: Dat was a merge going wrong. :)

**Review by denschub — CHANGES_REQUESTED** [review]
Looks fine, but there's a conflict marker left :)

File: _routes/post_interactions.md (hunk 1 above)
Comment (jhass) [reply _routes/post_interactions.md]: dammit

Comment (jhass) [reply _routes/post_interactions.md]: I wholeheartedly blame sublime for this one
Screenshot from 2020-02-02 18-58-15

Comment (skyschub) [reply _routes/post_interactions.md]: Wow.. that's something, fore sure.

**Review by denschub — APPROVED** [review]

**Pull request merged**